crates/parry3d/Cargo.toml

```diff
@@ -89,6 +89,11 @@ oorandom = "11"
 ptree = "0.4.0"
 rand = { version = "0.8" }
 macroquad = "0.4.12"
+nalgebra = { version = "0.33", default-features = false, features = [
+    "libm",
```

Comment:
Is the `libm` feature really necessary for the benchmarks?

Reply:
it's not, I'm removing it.